google/cloud/aiplatform/metadata/schema/system_schema.py

```diff
@@ -0,0 +1,508 @@
+        metadata: Optional[Dict] = None,
+        state: gca_artifact.Artifact.State = gca_artifact.Artifact.State.LIVE,
+        **kwargs,
+    ):
+        """Args:
+        metrics_name (str):
+            Optional. The resource name of the Artifact following the format as follows.
+            This is globally unique in a metadataStore:
+            projects/123/locations/us-central1/metadataStores/<metadata_store_id>/artifacts/<resource_id>.
+        accuracy (float):
+            Optional. Defaults to zero.
+        precision (float):
+            Optional. Defaults to zero.
+        recall (float):
+            Optional. Defaults to zero.
+        f1score (float):
+            Optional. Defaults to zero.
+        mean_absolute_error (float):
+            Optional. Defaults to zero.
+        mean_squared_error (float):
+            Optional. Defaults to zero.
+        uri (str):
+            Optional. The uniform resource identifier of the artifact file. May be empty if there is no actual
+            artifact file.
+        display_name (str):
+            Optional. The user-defined name of the base.
+        schema_version (str):
+            Optional. schema_version specifies the version used by the base.
+            If not set, defaults to use the latest version.
+        description (str):
+            Optional. Describes the purpose of the Artifact to be created.
+        metadata (Dict):
+            Optional. Contains the metadata information that will be stored in the Artifact.
+        state (google.cloud.gapic.types.Artifact.State):
+            Optional. The state of this Artifact. This is a
+            property of the Artifact, and does not imply or
+            capture any ongoing process. This property is
+            managed by clients (such as Vertex AI
+            Pipelines), and the system does not prescribe or
+            check the validity of state transitions.
+        **kwargs:
+            Optional. Additional Args that will be passed directly to the Artifact base method for backward compatibility.
+        """
+        extended_metadata = metadata or {}
+        extended_metadata["accuracy"] = accuracy
+        extended_metadata["precision"] = precision
+        extended_metadata["recall"] = recall
+        extended_metadata["f1score"] = f1score
+        extended_metadata["mean_absolute_error"] = mean_absolute_error
+        extended_metadata["mean_squared_error"] = mean_squared_error
+        extended_metadata["resourceName"] = metrics_name
+
+        super(Metrics, self).__init__(
+            schema_title=self.SCHEMA_TITLE,
+            resource_name=metrics_name,
+            uri=uri,
+            display_name=display_name,
+            schema_version=schema_version,
+            description=description,
+            metadata=extended_metadata,
+            state=state,
+            kwargs=kwargs,
+        )
+
+
+class ContainerExecution(base_execution.BaseExecutionSchema):
```

Comment:
Consider splitting the artifact and execution schemas into different modules.

Reply:
Done, there is one question on naming I'm currently naming them google_artifact_schema, system_artifact_schema and system_execution_schema. One point that @jialuzh  raised is we could move the modules to sub folders for google and system to have an import statement as follows" 

```python 
from  google.cloud.aiplatform.metadata.schema import system_artifact_schema

# instead of 

from  google.cloud.aiplatform.metadata.schema.system import artifact_schema
```

they only catch is if users need to use by artifact types from google and system then they will need to alias it as system_artifact_schema and google_artifact_schema which defeats the above simplification. I will add this as a comment to the bug bash to get feedback on if one is preferred over the other.